tests/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if(GMOCK_FOUND)
     cuke_add_test(integration/HookRegistrationTest)
     cuke_add_test(integration/StepRegistrationTest)
     cuke_add_test(integration/TaggedHookRegistrationTest)
-    if (NOT MSVC)
+    if(NOT WIN32)
```

Comment:
Have you tried running this with MinGW? Because I don't think there's anything to this test that should prevent it from running on Windows. (Just asking because I'm only noticing that this test is disabled for Windows just now).